<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managementCostUi</class>
 <widget class="QWidget" name="managementCostUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="header">
     <property name="title">
      <string>Header</string>
     </property>
     <layout class="QGridLayout" name="headerLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_seller">
        <property name="text">
         <string>Seller</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="simpleCombo" name="comboSeller"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_date">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="date">
         <date>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_number">
        <property name="text">
         <string>Reference</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="line_number"/>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioPurchasing">
        <property name="text">
         <string>Management</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>185</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioGoods">
        <property name="text">
         <string>Food &amp;&amp; Beverage</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnGo">
        <property name="text">
         <string>Go!</string>
        </property>
        <property name="icon">
         <iconset resource="../../../core/res.qrc">
          <normaloff>:/conferma24x24.png</normaloff>:/conferma24x24.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="row">
     <property name="title">
      <string>Row</string>
     </property>
     <layout class="QGridLayout" name="rowLayout">
      <item row="0" column="0">
       <widget class="simpleTable" name="tableView">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>7</number>
     </property>
     <property name="midLineWidth">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>$</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="toolLayout"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>simpleTable</class>
   <extends>QTableView</extends>
   <header location="global">simpletable.h</header>
  </customwidget>
  <customwidget>
   <class>simpleCombo</class>
   <extends>QComboBox</extends>
   <header location="global">simplecombo.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../core/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnGo</sender>
   <signal>clicked()</signal>
   <receiver>managementCostUi</receiver>
   <slot>goPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goPressed()</slot>
 </slots>
</ui>
